swc-uxp-starter/README.md

```diff
@@ -0,0 +1,132 @@
+# **UXP Plugin with Spectrum Web Component**
+Since UXP v7.0
+# **Objective**
+This sample will help you to create a Spectrum Web Component (SWC) based UXP plugin. It includes setting up the project, integrating the SWC components, and mounting it over the host applications - Photoshop and XD.
+
+Start by learning about [Spectrum Web Components](https://opensource.adobe.com/spectrum-web-components/index.html)
+
+
+# **Getting started**
+1. Start by installing the dependencies `yarn install`
+2. Prepare the bundle using Webpack `yarn build`. You will notice a **dist** folder after this step.
+3. Add the plugin in UXP Developer Tool (UDT). Click on 'Add Plugin' in UDT and navigate to the `manifest.json` inside the dist folder.
+4. Load the plugin in Photoshop/XD by launching the app. Add the plugin in the UDT.Click on the plugin action menu in UDT, select More > Advanced and enter the relative location for the plugin builder folder (./dist is this case).
```

Comment:
Update pt #3 based on this. You want the developers to use the manifest.json (outside the dist folder) to load into UDT and configure the dist folder using the Advance option.